tests/scripts/github_tag_teams.py

```diff
@@ -122,7 +122,7 @@ def add_tag(tag, users):
     for tag in result:
         result[tag] = list(set(result[tag]))
 
-    return {k.lower(): v for k, v in result.items()}
+    return {k.lower(): v for k, v in result.items() if k.strip() != ""}
```

Comment:
Since empty strings/collections are false-y, the `if k.strip() != ""` can be replaced with just `if k.strip()`.